test/collection_test.js

```diff
@@ -157,6 +159,45 @@ describe("Collection", () => {
       });
     });
 
+    describe("hooks registration", () => {
+      function registerHooks(hooks) {
+        return new Collection(TEST_BUCKET_NAME, TEST_COLLECTION_NAME, api, {
+          hooks,
+          adapter: IDB
```

Comment:
I suspect that's because we're never expected to use Collection directly, only through the Kinto instance... Ok leave it that way, no big deal.